modules/acquia_cms_common/src/Services/AcmsUtilityService.php

```diff
@@ -62,7 +62,10 @@ public function rebuildSiteStudio() {
     $batch = &batch_get();
     $batch = NULL;
     unset($batch);
-    return DX8CommandHelpers::rebuild([]);
+    return DX8CommandHelpers::rebuild([
+      'verbose' => '',
```

Comment:
@chandan-singh7929 Don't we need to show verbose message anymore now ??

Reply:
I don't think we need to from here at least.